### DISCUSSION TITLE
Open IPTV.  Designed for couch XBOX IPTV App.

### DISCUSSION THREAD

**Discussion by tarrazuapps** [iptv-org/community#260] [Show and tell]
Although I know the ONE device for all your entertainment dream of the xbox one is dead, I really like the idea of having just 1 device connected to my tv that does everything, including gaming.  I do have a cable box, but I hate to have to use it and switching remotes when all I do is occasionally  watch my local news or some local shows that are just over the air tv. So I looked for options, including converters like the HD Home Run; but that really wont work in my country since we use a different digital standard than the states. Then I figured the channels I wanted were already being broadcasted over the internet.

With the goal of not having to switch remotes to watch my local channels, I been working for some time in an IPTV client for XBOX. I know there are already a bunch out there; but I tried them out and I found the navigation of most of them cumbersome. I wanted something simpler focused on the "zapping", couch experience and that was built around using a remote.
I also know you can use Kodi with with a plugin, but then again the navigation experience was not quite what I wanted.
In addition to that I built a couple of features, like trying to get the stream url from a website html since some sites still have their streams open, but will change them from time to time.

If somebody would like to try it out and give me your thoughts, I would really appreciate it. This app will be free and wont have ads. 

The application is now available. You can get it here:  https://www.microsoft.com/en-us/p/open-iptv/9n9gc8l5mldm 

Here are a couple of screenshots. If you want to try it out, let me know and I will send you the link.
![Watch1](https://user-images.githubusercontent.com/86750752/124068852-8156d380-d9f8-11eb-89f3-3d0b0e5e7251.PNG)

![Watch2](https://user-images.githubusercontent.com/86750752/124068869-87e54b00-d9f8-11eb-8910-8607a9854010.PNG)

![search](https://user-images.githubusercontent.com/86750752/124068881-8c116880-d9f8-11eb-8ea3-cb549eb22f31.PNG)

![config1](https://user-images.githubusercontent.com/86750752/124068896-92074980-d9f8-11eb-9c63-ad478de3e921.PNG)

![config2](https://user-images.githubusercontent.com/86750752/124068910-96336700-d9f8-11eb-93fd-88df4ed0f992.PNG)

**Comment by tarrazuapps** [comment]
If this could get posted in the AwesomeIPTV readme as a resource, I would really appreciate it.

Thank you.

**Reply by BellezaEmporium** [reply]
I see that it's also available for me as an UWP app. I'm going to have a try on this and tell you.

EDIT : Nevermind, the store isn't consistent when it talks about "app availability". I don't have an XBox One to test this sadly, but i'll pass the message to add it onto Awesome IPTV.

**Reply by tarrazuapps** [reply]
Yeah, I tried to make it work well in windows as well, but I do not have the time to put all the work it needs to properly make the interface adapt well in both.

Thanks a lot.
